### PR TITLE
Migrate core/store/storage api urls

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Migrate core/store/storage api urls

Export config for storage selector

[stage-18]

## Motivation and Context
Angular migration.

## How Has This Been Tested?
Tested locally, tests pass, tested stage.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: src/scripts/common-header/config/config.js [hunk 1]
```diff
@@ -21,15 +21,15 @@
     .constant('LOCALES_SUFIX', '.json');
 
   angular.module('risevision.common.config')
+    .value('environment', {
```
Comment (ezequielc) [inline src/scripts/common-header/config/config.js:24]: @alex-deaconu I am curious, why do we need to keep these variables here? Is it for unit tests? If so, should we merge with `config/test.js`? Thanks

Comment (alex-deaconu) [reply src/scripts/common-header/config/config.js:24]: @ezequielc this is the the backup config for the common-header build. Thanks!

Comment (ezequielc) [reply src/scripts/common-header/config/config.js:24]: Got it, thanks!

**Review by ezequielc — APPROVED** [review]

**Pull request merged**